content/install_maintain/installation/installation-modes.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: bt_wiki
+title: Understanding the Cloudify platform deployment modes
+description: Understanding the Cloudify platform deployment modes.
+category: Installation
+draft: false
+weight: 5
+---
+
+Before you deploy your Cloudify platform, please use this page to get familiar with the different deployment options for the Cloudify Platform and select the mode that suits you best.
+
+Cloudify offers the following modes of installation:
+
+* All-in-one single box deployment
+* Cloudify Compact cluster (3 nodes)
+* Cloudify fully distributed cluster (9+ nodes)
```

Comment:
I think it would be better to write "9 nodes" instead of "9+ nodes". We don't test the option of more than 9 nodes.